docs/application/native/tutorials/feature/app-graphics-animation.md

```diff
@@ -147,9 +147,9 @@ To implement animations in an application:
     `ecore_animator_pos_map()` function. For the second parameter, enter
     the style of the animation. The style types are described in the
     `_Ecore_Pos_Map` enumeration (in
-    [mobile](../../api/mobile/latest/group__Ecore__Animator__Group.html#ga2db0d0f0f3973829c7f700e5af3e041c)
+    [mobile](../../api/mobile/latest/group__Ecore__Animator__Group.html)
     and
-    [wearable](../../api/wearable/latest/group__Ecore__Animator__Group.html#ga2db0d0f0f3973829c7f700e5af3e041c) applications)
+    [wearable](../../api/wearable/latest/group__Ecore__Animator__Group.html) applications)
```

Comment:
Just a suggestion. How about changing to "#ga1047011623d9e2ea31b7e5678b451016"? Or is it better to remove?